gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageWriteChannel.java

```diff
@@ -141,4 +146,15 @@ public void handleResponse(StorageObject response) {
   public GoogleCloudStorageItemInfo getItemInfo() {
     return this.completedItemInfo;
   }
+
+  /**
+   * Sets kmsKeyName used in the object create request for this channel.
+   *
+   * Use before calling ``initialize``.
+   *
+   * @param kmsKeyName Name of Cloud KMS key to use to encrypt the newly created object.
+   */
+  public void setKmsKeyName(String kmsKeyName) {
```

Comment:
Why setter is necessary?

Reply:
This was necessary when you couldn't set this value from the constructor, but you can now.